Document how to run local dev (#30)
CrowdStrike/community: README.md

On that webpage, you can find content such as:
- [List of CrowdStrike's Open Source Projects](https://crowdstrike.github.io/community/)
- [Guidance for CrowdStrike Engineering](https://crowdstrike.github.io/community/open-source-policy/#engineering)
- [Guidance for All Other Departments/Orgs](https://crowdstrike.github.io/community/open-source-policy/#everyone-else)

> ## Developing the Site

Reviewer: ```suggestion
## Building

To build from the repo after cloning:
```

Author: `build` eludes to a one time thing, like a compilation. Prefer "Developing" since it's more of a continuous process.

1. Ensure ``bundler`` and `jekyll` and ``rake`` are installed locally:
```shell
gem install bundler jekyll rake
```

> 2. Install NPM dependencies. The webroot is the `docs` folder, so you must change into that directory:
> ```shell
> cd docs/ && npm install
> ```

Reviewer: ```suggestion
2. Change directory to the `docs` folder and use `bundler` to build the site:
```shell
cd docs/ && bundle exec jekyll build
```
```

Author: If you don't `npm install` then the site will fail to build (many missing modules)

Reviewer: Odd.... I haven't seen that behavior at all. I don't even have npm installed. Is this a macos requirement?

Author: Dunno. I do develop on OSX, and if the node modules are missing or outdated, a bunch of errors are thrown and the site doesn't build locally.

Reviewer: funny... okay. I'll merge and we can fix later.

Author: ( I can't duplicate anymore, since, well, they're all installed now)

> 3. Start the development server locally. It will watch for chances you make, and reload (nearly) instantly:

Reviewer: ```suggestion
3. Start the development server locally. Add the `--incremental` flag if you want to watch and reload changes:
```

Author: `bundle exec jekyll serve` automatically adds the ``--incremental`` flag

Reviewer: From what I have seen, it doesn't get turned on automatically. Line 32 shows it as disabled.

Should we have instructions for linux vs macos?
